app/services/acuant/liveness.rb

```diff
@@ -25,18 +33,18 @@ def fetch_face_from_document
       Base64.strict_encode64(data)
     end
 
-    def check_face_match(image1, image2)
-      data = wrap_network_errors { facematch_service.facematch(facematch_body(image1, image2)) }
-      return unless data
-      facematch_pass?(data)
+    def selfie_face_matches_document?(img1, img2)
+      @results = wrap_network_errors { facematch_service.facematch(facematch_body(img1, img2)) }
+      facematch_pass?
     end
 
-    def facematch_pass?(data)
-      data['IsMatch']
+    def facematch_pass?
+      results&.dig('IsMatch')
     end
 
-    def selfie_live?(data)
-      data['LivenessResult']['LivenessAssessment'] == 'Live'
+    def liveness_result_returns_live?
+      results&.dig('LivenessResult') &&
+        results['LivenessResult']['LivenessAssessment'] == 'Live'
```

Comment:
`#dig` can take multiple arguments, both these lines can be combined:

```
results&.dig('LivenessResult', 'LivenessAssessment') == 'Live'
```

Reply:
i like!